### ISSUE TITLE
Floating Action Button Not Working on Android.

### ISSUE THREAD

**Issue by juanbonoso** [Akylas/nativescript-material-components#136]
### Which platform(s) does your issue occur on?
Android in all versions. 

### Please, provide the following version numbers that your issue occurs with:

- CLI: 6.6.0-2020-03-12-100308-14143
- Cross-platform modules: @nativescript/core@6.6.0-next-2020-04-10-164516-02
- Runtime(s): 
```
"tns-ios": {
			"version": "6.5.0-2020-03-12-133513-01"
		},
"tns-android": {
			"version": "6.5.0-2020-04-13-120923-01"
		}

```
- Plugin(s): 

"nativescript-material-floatingactionbutton": "^3.1.17"

### Please, tell us how to recreate the issue in as much detail as possible. 
Describe the steps to reproduce it.

Just install FAB, and try to use it in android. it will give the following error:

```
System.err: An uncaught Exception occurred on "main" thread.
System.err: Calling js method onCreateView failed
System.err: TypeError: nativeView.getTransformationMethod is not a function
System.err: 
System.err: StackTrace:
System.err: TextBase.initNativeView(file: node_modules/@nativescript/core/ui/text-base/text-base.android.js:79:55)
System.err:     at Button.initNativeView(file: node_modules/@nativescript/core/ui/button/button.android.js:55:40)
System.err:     at ViewBase.setNativeView(file: node_modules/@nativescript/core/ui/core/view-base/view-base.js:567:17)
System.err:     at ViewBase._setupUI(file: node_modules/@nativescript/core/ui/core/view-base/view-base.js:546:13)
System.err:     at (file: node_modules/@nativescript/core/ui/core/view-base/view-base.js:553:18)
System.err:     at LayoutBaseCommon.eachChildView(file: node_modules/@nativescript/core/ui/layouts/layout-base-common.js:125:25)
System.err:     at ViewCommon.eachChild(file: node_modules/@nativescript/core/ui/core/view/view-common.js:925:13)
System.err:     at ViewBase._setupUI(file: node_modules/@nativescript/core/ui/core/view-base/view-base.js:552:13)
System.err:     at (file: node_modules/@nativescript/core/ui/core/view-base/view-base.js:553:18)
System.err:     at ContentView.eachChildView(file: node_modules/@nativescript/core/ui/content-view/content-view.js:70:12)
System.err:     at PageBase.eachChildView(file: node_modules/@nativescript/core/ui/page/page-common.js:126:39)
System.err:     at ViewCommon.eachChild(file: node_modules/@nativescript/core/ui/core/view/view-common.js:925:13)
System.err:     at ViewBase._setupUI(file: node_modules/@nativescript/core/ui/core/view-base/view-base.js:552:13)
System.err:     at ViewBase._addViewCore(file: node_modules/@nativescript/core/ui/core/view-base/view-base.js:448:17)
System.err:     at ViewBase._addView(file: node_modules/@nativescript/core/ui/core/view-base/view-base.js:437:13)
System.err:     at FragmentCallbacksImplementation.onCreateView(file: node_modules/@nativescript/core/ui/frame/frame.android.js:700:18)
System.err:     at FragmentClass.onCreateView(file: node_modules/@nativescript/core/ui/frame/fragment.android.js:29:37)
System.err:     at com.tns.Runtime.callJSMethodNative(Native Method)
System.err:     at com.tns.Runtime.dispatchCallJSMethodNative(Runtime.java:1286)
System.err:     at com.tns.Runtime.callJSMethodImpl(Runtime.java:1173)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1160)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1138)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1134)
System.err:     at com.tns.FragmentClass.onCreateView(FragmentClass.java:53)
System.err:     at androidx.fragment.app.Fragment.performCreateView(Fragment.java:2600)
System.err:     at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManagerImpl.java:881)
System.err:     at androidx.fragment.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManagerImpl.java:1238)
System.err:     at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManagerImpl.java:1303)
System.err:     at androidx.fragment.app.BackStackRecord.executeOps(BackStackRecord.java:439)
System.err:     at androidx.fragment.app.FragmentManagerImpl.executeOps(FragmentManagerImpl.java:2079)
System.err:     at androidx.fragment.app.FragmentManagerImpl.executeOpsTogether(FragmentManagerImpl.java:1869)
System.err:     at androidx.fragment.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManagerImpl.java:1824)
System.err:     at androidx.fragment.app.FragmentManagerImpl.execPendingActions(FragmentManagerImpl.java:1727)
System.err:     at androidx.fragment.app.FragmentManagerImpl$2.run(FragmentManagerImpl.java:150)
System.err:     at android.os.Handler.handleCallback(Handler.java:789)
System.err:     at android.os.Handler.dispatchMessage(Handler.java:98)
System.err:     at android.os.Looper.loop(Looper.java:164)
System.err:     at android.app.ActivityThread.main(ActivityThread.java:6541)
System.err:     at java.lang.reflect.Method.invoke(Native Method)
System.err:     at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
System.err:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)

```

### Is there any code involved? 
```
<MDFloatingActionButton
				rippleColor="blue"
				width="50"
				height="50"
				text="HO"
			/>
```

**Comment by farfromrefug** [comment]
@juanbonoso it is an issue with N.
You should be careful when using nighlty builds of N, it has been fixed .
Next time try with the official build first ;)

**Comment by juanbonoso** [comment]
Thanks for the feedback, i updated everything to use official build.

Now it looks like this:

- CLI: 6.5.0
- Cross-platform modules: 6.5.1
- Runtime(s): 
```
"tns-ios": {
			"version": "6.5.0"
		},
"tns-android": {
			"version": "6.5.0"
		}
```
- Plugin(s): 
		"nativescript-material-button": "^3.2.1",
		"nativescript-material-cardview": "^3.2.1",
		"nativescript-material-dialogs": "^3.2.1",
		"nativescript-material-snackbar": "^3.2.1",
		"nativescript-material-floatingactionbutton": "^3.2.1"

I got the same error i have mentioned before; have deleted and re-installed node_modules , platforms, hooks to avoid any errors with N. Here's a capture to show that only works on iOS.

![Screen Shot 2020-04-14 at 13 54 23](https://user-images.githubusercontent.com/43289624/79262898-9842f180-7e57-11ea-9b77-de92fdd1a456.png)

**Comment by farfromrefug** [comment]
seems like you are actually using a "dev" version of N. This is was recently fixed on N side

**Comment by juanbonoso** [comment]
I can assure you, that i am not using a dev version of N. I have the last stable release from nativescript(6.5).
May be a problem with vue? nativescript-vue? or the problem is being solved in nativescript 6.6?
